spec/lutaml/model/namespace_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+module NamespaceSpec
+  class NestedChild < Lutaml::Model::Serializable
+    attribute :name, :string
+
+    xml do
+      root "NestedChild"
+
+      map_element :name, to: :name
+    end
+  end
+
+  class Child < Lutaml::Model::Serializable
+    attribute :nested_child, NestedChild
+
+    xml do
+      root "NestedChild"
+
+      map_element :NestedChild, to: :nested_child
+    end
+  end
+
+  class Parent < Lutaml::Model::Serializable
+    attribute :child, Child
+
+    xml do
+      root "Parent"
+      namespace "https://abc.com"
```

Comment:
In the test we should test additional nesting levels and when the namespaces change in between. If the parent and child both have the same namespace, what happens? Need to update documentation. Going to approve and merge this first.